contracts/NonfungiblePositionManager.sol

```diff
@@ -76,25 +66,23 @@ contract NonfungiblePositionManager is INonfungiblePositionManager, BaseLiquidit
         returnData = new bytes[](actions.length);
         for (uint256 i; i < actions.length; i++) {
             if (actions[i] == Actions.INCREASE) {
-                (uint256 tokenId, uint256 liquidity, bytes memory hookData, bool claims) =
-                    abi.decode(params[i], (uint256, uint256, bytes, bool));
-                returnData[i] = abi.encode(increaseLiquidity(tokenId, liquidity, hookData, claims, sender));
+                (uint256 tokenId, uint256 liquidity, bytes memory hookData) =
+                    abi.decode(params[i], (uint256, uint256, bytes));
+                returnData[i] = abi.encode(increaseLiquidity(tokenId, liquidity, hookData, sender));
             } else if (actions[i] == Actions.DECREASE) {
-                (uint256 tokenId, uint256 liquidity, bytes memory hookData, bool claims) =
-                    abi.decode(params[i], (uint256, uint256, bytes, bool));
-                returnData[i] = abi.encode(decreaseLiquidity(tokenId, liquidity, hookData, claims, sender));
+                (uint256 tokenId, uint256 liquidity, bytes memory hookData) =
+                    abi.decode(params[i], (uint256, uint256, bytes));
+                returnData[i] = abi.encode(decreaseLiquidity(tokenId, liquidity, hookData, sender));
             } else if (actions[i] == Actions.MINT) {
                 (LiquidityRange memory range, uint256 liquidity, uint256 deadline, address owner, bytes memory hookData)
                 = abi.decode(params[i], (LiquidityRange, uint256, uint256, address, bytes));
-                (BalanceDelta delta, uint256 tokenId) = mint(range, liquidity, deadline, owner, hookData, sender);
-                returnData[i] = abi.encode(delta, tokenId);
+                returnData[i] = abi.encode(mint(range, liquidity, deadline, owner, hookData));
```

Comment:
why is there only a deadline on mint? imo this should go on the high level call and not in these commands - to prevent duplication and to separate it from the internals
like
```
function modifyLiquidities(bytes calldata unlockData, uint256 deadline) checkDeadline(deadline) public ...
```

and then you dont need it anywhere else